### DISCUSSION TITLE
OA crashes when trying to connect to Audible (macOS Ventura Beta)

### DISCUSSION THREAD

**Discussion by raphaelmeyhoefer** [openaudible/openaudible#1037] [General]
I installed the beta of macOS Ventura and OpenAudible is not working properly there.

Whenever I want to switch between Audible Accounts the software crashes (before I can enter my credentials in this tiny in-app-browser). I believe it has something to do with the macOS beta but maybe someone else can verify or you guys are working on it. It happens not only when trying to log in into audible but every time I try opening the in-app-browser, even the "project website"-link.

I have tried both my old license (up to version 3.0.2) and my new license with the most recent version. Both do crash.

Unfortunately I cannot get a bug report produced. So I cannot share more information, I might have read for a split second that this problem might have something to do with Java.

Maybe someone has a fix for that (I did not manage to set OA up with an external browser).

Thant You!

**Comment by openaudible** [comment]
Someone just posted a bug and we're trying to get a bug report... #1050

**Reply by openaudible** [reply]
Looking for one good bug report from the Terminal to get this fixed. Unfortunately unable to reproduce after installing Ventura.

**Comment by oboe45** [comment]
For what its worth, I have installed Ventura release on Intel and Apple
chip sets on current systems and no problems with either installation using
Current Openaudible

**Comment by openaudible** [comment]
We have removed the OpenAudible internal browser from 3.6.3.x beta and will roll out that change for 3.7 shortly. 
We also determined what the problem was-- so even if you choose the browser login method for 3.7, it should still work on Intel Macs.
Thanks all for the reports.

**Comment by openaudible** [comment]
https://openaudible.org/beta
